### ISSUE TITLE
Chinese characters displayed in wasm are garbled characters

### ISSUE THREAD

**Issue by allenlu2008** [slint-ui/slint#3460]
slint 1.2.0
In the local exe, Chinese characters are displayed normally, but after compiling to wasm, Chinese characters are displayed as garbled characters.
wasm:
![20230914101754.png](https://s2.loli.net/2023/09/14/Rj4iqgWNJM16s3c.png)
but local exe :
![20230914102028.png](https://s2.loli.net/2023/09/14/iN8xnlIJPZBuke9.png)

**Comment by tronical** [comment]
Yes indeed, this is a limitation of our wasm builds at the moment. One way of working around it would be to import a `.ttf` font that provides coverage manually (`import "noto-sans.ttf"` where the `.ttf` is one of the Noto variants with CJK coverage) and then set it as `default-font-family` on the `Window` element. That way the font is embedded in the wasm binary and it should render with it.

**Comment by allenlu2008** [comment]
I have imported the local ttf font file into wasm, but still cannot display Chinese in wasm. The code is as follows:
import { LineEdit , Button} from "std-widgets.slint";

import "./msyh.ttf";

export component mylogin inherits Window {

    default-font-family: "./msyh.ttf";

    width: 480px; height:320px;

    title: "系统登录";

    icon: @image-url("../img/icon2.png");

    background: @linear-gradient(60deg,#dddddd 0%,#c5d4dd 50%,#c5ed97 100%);
}

The size of the generated file already includes the size of the font file

![image](https://github.com/slint-ui/slint/assets/5217796/360867ac-afc7-4b8e-822d-7db3d11a30e1)

But it's still invalid and still messy code

![image](https://github.com/slint-ui/slint/assets/5217796/041bb56f-b8d4-42f1-a6ab-c6a20743cdeb)

**Comment by ogoffart** [comment]
> default-font-family: "./msyh.ttf";

I think you need to use the name of the font and not his filename.
